### PR TITLE
notify all PRs on manual system tests

### PR DESCRIPTION
**Pull request by stewartie4**
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/blobber/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master BEFORE merging this PR  
  
Associated PRs (Link as appropriate):
0chain: https://github.com/0chain/0chain/pull/835
blobber: https://github.com/0chain/blobber/pull/478
zwalletcli: https://github.com/0chain/zwalletcli/pull/80
zboxxli: https://github.com/0chain/zboxcli/pull/150
actions: https://github.com/0chain/actions/pull/8

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | feature/test-branch  |
| gosdk  | feature/notify-pr-gosdk  |
| 0boxcli  | feature/notify-pr-zboxcli  |
| 0walletcli  | feature/notify-pr-zwalletcli  |
| 0chain  | feature/notify-pr  |
| blobber  | feature/notify-pr-blobber  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| zproxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | feature/test-branch  |
| gosdk  | feature/notify-pr-gosdk  |
| 0boxcli  | feature/notify-pr-zboxcli  |
| 0walletcli  | feature/notify-pr-zwalletcli  |
| 0chain  | feature/notify-pr  |
| blobber  | feature/notify-pr-blobber  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| zproxy  | staging  |
| 0search  | staging  |

File: .github/workflows/system_tests.yml [hunk 1]
```diff
@@ -1,138 +1,72 @@
...
-        message: "manual tests success"
-        system_tests_branch: ${{ github.event.inputs.system_tests_branch }}
-        zbox_cli_branch: ${{ github.event.inputs.zbox_cli_branch }}
-        zwallet_cli_branch: ${{ github.event.inputs.zwallet_cli_branch }}
-        gosdk_branch: ${{ env.CURRENT_BRANCH_HEAD }}
-        zerochain_branch: ${{ github.event.inputs.miner_image }}
-        blobber_branch: ${{ github.event.inputs.blobber_image }}
-        github_token: ${{ secrets.SVC_ACCOUNT_SECRET }}
+  system-tests:
+    uses: 0chain/actions/.github/workflows/manual_system_tests.yml@feature/notify-PR-convenience
```
Comment (cnlangzi) [inline .github/workflows/system_tests.yml]: we will always use `feature/notify-PR-convenience` instead of `master`?

Comment (stewartie4) [reply .github/workflows/system_tests.yml]: nope, once https://github.com/0chain/actions/pull/8 has merged to master we can change this to master

**Review by cnlangzi — DISMISSED** [review]
LGTM

**Review by cnlangzi — APPROVED** [review]

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | staging  |
| 0boxcli  | staging  |
| 0walletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Pull request merged**